Kernel/FileSystem/BlockBasedFileSystem.cpp

```diff
@@ -60,6 +60,10 @@ class DiskCache {
             return nullptr;
         auto& entry = const_cast<CacheEntry&>(*it->value);
         VERIFY(entry.block_index == block_index);
+        if (!entry_is_dirty(entry)) {
```

Comment:
if itʼs not first, right? (not sure if itʼs worth checking)

Reply:
I don't think that is a common scenario but it is not a bad idea to check if it was first. I will change it.